src/main/java/uk/gov/hmcts/reform/migration/service/migrate/DefaultPanelCompositionMigration.java

```diff
@@ -58,18 +62,42 @@ public List<SscsCaseDetails> fetchCasesToMigrate() {
         } else {
             return repository.findCases(searchQuery, true)
                 .stream()
-                .filter(caseDetails -> READY_TO_LIST.toString().equals(caseDetails.getState())
-                    && caseDetails.getData().getSchedulingAndListingFields().getHearingRoute()
-                    .equals(HearingRoute.LIST_ASSIST))
-                .toList();
+                .filter(caseDetails -> {
+                    var hearingRoute = caseDetails.getData().getSchedulingAndListingFields().getHearingRoute();
+                    var panelComposition = caseDetails.getData().getPanelMemberComposition();
+                    return READY_TO_LIST.toString().equals(caseDetails.getState())
+                        && LIST_ASSIST.equals(hearingRoute)
+                        && (isNull(panelComposition) || panelComposition.isEmpty());
+                }).toList();
         }
     }
 
     @Override
     public UpdateResult migrate(CaseDetails caseDetails) {
         if (caseDetails.getState().equals(READY_TO_LIST.toString())) {
-
             String caseId = caseDetails.getId().toString();
+            var data = caseDetails.getData();
+            if (nonNull(data)) {
```

Comment:
not sure we need this since we're already filtering them out in the fetchCasesToMigrate method